packages/caliper-fabric/lib/fabricNetwork.js

```diff
@@ -79,16 +75,16 @@ class FabricNetwork {
                 const cObj = this.network.clients[client].client;
                 // normalize paths
                 if (cObj.credentialStore) {
-                    cObj.credentialStore.path = CaliperUtils.resolvePath(cObj.credentialStore.path, workspaceRoot);
-                    cObj.credentialStore.cryptoStore.path = CaliperUtils.resolvePath(cObj.credentialStore.cryptoStore.path, workspaceRoot);
+                    cObj.credentialStore.path = CaliperUtils.resolvePath(cObj.credentialStore.path, this.workspaceRoot);
```

Comment:
Actually, `resolvePath` uses the configured workspace as the default value for a while now, so the `this.workspaceRoot` arguments are unnecessary. Since it's only used in `resolvePath`, it can be removed from the class altogether.
(I know it wasn't introduced by this PR, but it's a good place as any to clear it up 😄 )